### ISSUE TITLE
[FEATURE REQUEST] Remove forced landscape mode

### ISSUE THREAD

**Issue by HarriBuh** [sz3/cfc#23]
Please let us choose which camera mode to pick, either landscape OR portrait. I find it rather confusing and uncomfortable to always use landscape mode when taking the picture.

**Comment by sz3** [comment]
I agree this would be nice, and I feel your pain -- I kind of hate how it's locked to landscape too. Unfortunately, I don't think the fix for this is small (at least, it won't be easy for me), and there's a lot of other things I have to work on. :disappointed:

**Comment by Anonymous3-a** [comment]
Just saying, cfc still does work in portrait mode just fine (albeit everything looks rotated 90 degrees), so if all you want is holding your phone that way, it works.
